spec/lib/oidc/key_service_spec.rb

```diff
@@ -23,27 +23,64 @@
 
   describe '::get_key' do
     after do
-      described_class.instance_variable_set(:@current_key, {})
+      described_class.reset!
     end
 
     it 'returns the current key if it already exists' do
-      described_class.current_keys['key'] = 'key'
+      described_class.instance_variable_set(:@current_keys, 'key' => 'key')
```

Comment:
Thanks for cleaning up the public API of this class. I much prefer the test suite overriding the protection of the instance variable to the public API of the class exposing it's state _just_ to enable a testing scenario.